batch7crud-roll3/src/main/java/com/lftechnology/batch7crud/validator/StudentValidator.java

```diff
@@ -11,8 +11,7 @@
 public class StudentValidator extends ValidatorUtil implements Validator<Student> {
 
   @Override
-  public Student createObject(Map inputs) throws ValidationException {
-    Student student = new Student();
+  public void validateInputs(Map inputs) throws ValidationException {
     Map<String, String> errors = new HashMap<>();
     String roll = (String) inputs.get(ROLL);
     String name = (String) inputs.get(NAME);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "name" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1481)